src/controllers/userProfileHandler.js

```diff
@@ -130,8 +136,73 @@ class UserProfileController {
       });
     }
   }
+
+  /**
+   * @description - This method is responsible for retrieving all profiles of users on the patform
+   *
+   * @static
+   * @param {object} request - Request sent to the router
+   * @param {object} response - Response sent from the controller
+   *
+   * @returns {object} - object representing response message and user object
+   *
+   * @memberof UserProfileController
+   */
+  static async getAllProfiles(request, response) {
+    const { payload } = request.userData;
+    const { username } = payload;
+    try {
+      let allProfiles = await User.findAll({
+        where: {
+          username: {
+            $notLike: `${username}`
```

Comment:
What does `$notLike` do in the query?

Reply:
@okoroemeka nice observation, I'm using that return just the users with usernames that aren't like the username of the logged in user. So it returns every other user profile except the person logged in